### PR TITLE
for peer review

### PR DESCRIPTION
**Pull request by Phil-Sparkes**

### CODE REVIEW

File: Pixelels/Main.py [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+            Blue = 255 - Blue
+
+            PXArray[X, Y] = (Red, Green, Blue)
+
+#Greyscale
+def GreyScale():
+
+    for Y in xrange(Height):
+        for X in xrange(Width):
+            Red = Screen.get_at((X, Y)).r
```
Comment (CallumPank) [inline Pixelels/Main.py:49]: Variables need to be lowercase

File: Pixelels/Main.py [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+            Green = Screen.get_at((X, Y)).g
+            Blue = Screen.get_at((X, Y)).b
+
+            PXArray[X, Y] = (Red/2, Green, Blue)
+
+#does weird stuff, dont trust this one
```
Comment (CallumPank) [inline Pixelels/Main.py:68]: Comments need to be aligned to side

File: Pixelels/Main.py [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+            Blue = Screen.get_at((X, Y)).b
+
+            if Red > 25 and Green > 25 and Blue > 25:
+                ColourChange = ColourChange + 1
+            else:
+                ColourChange = ColourChange - 1
+            Red = Red + ColourChange
+            Green = Green + ColourChange
+            Blue = Blue + ColourChange
+            if ColourChange > 150:
```
Comment (CallumPank) [inline Pixelels/Main.py:149]: Could simplify

File: Pixelels/Collage.py [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+PictureScale3 = 0
+ColourManip = 1
+
+
+Width = 1280   #Change these depending on the picture
+Height = 720
+
+Screen = pygame.display.set_mode((Width, Height))
+#Picture =pygame.transform.scale(Picture, (Width,Height))   #Scales the picture to fit the screen
+Picture =pygame.transform.scale(Picture, (PictureScale,PictureScale))     #Used for Collage
```
Comment (hannahlu) [inline Pixelels/Collage.py:24]: Use inline comments sparingly and try to have comments that explain blocks of code instead.

File: Pixelels/Collage.py [hunk 5]
```diff
@@ -0,0 +1,501 @@
+import pygame, sys, math
```
Comment (jack-maber) [inline Pixelels/Collage.py:1]: Packages should be imported on different lines according to PEP-8.

File: Pixelels/Collage.py [hunk 6]
```diff
@@ -0,0 +1,501 @@
...
+pygame.init()
+
+PictureScale = 80             # Used in the collage
+PictureScale2 = PictureScale
+PictureScale3 = 0
+ColourManip = 1
+
+
+Width = 1280   #Change these depending on the picture
+Height = 720
```
Comment (jack-maber) [inline Pixelels/Collage.py:20]: Constants should be in uppercase.

**Pull request closed without merging**